proto/activity/v1alpha/activity.proto

```diff
@@ -0,0 +1,75 @@
+syntax = "proto3";
+
+package com.mintter.activity.v1alpha;
+
+import "google/protobuf/timestamp.proto";
+
+option go_package = "mintter/backend/genproto/activity/v1alpha;activity";
+
+// ActivityFeed service provides information about the recent activity events happened in the system.
+service ActivityFeed {
+  // Lists the recent activity events,
+  // sorted by locally observed time (newest first).
+  rpc ListEvents(ListEventsRequest) returns (ListEventsResponse);
+}
+
+// The request to list the events.
+message ListEventsRequest {
```

Comment:
Don't we want to add some filtering here? so we only provide certain entity events or events related to a user?